src/en/howto-applicationupdateseries.md

```diff
@@ -0,0 +1,117 @@
+Title: Change an application's series pre-juju2.3
+
+#  Change an application's series pre-juju2.3
+
+## Overview
+
+When deploying an application with juju you can specify the series it
```

Comment:
I suggest:

When deploying an application with Juju you can specify the series it
will use, or use the default series for the model. Prior to Juju 2.3,
any units which are subsequently added to the application always use 
the series specified at deployment.

Reply:
updated